app/src/main/res/xml/actions.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version ="1.0" encoding ="utf-8"?>
+<actions>
+    <action intentName="actions.intent.OPEN_APP_FEATURE">
+        <fulfillment urlTemplate="https://com.mborowiec.antennapod/main{?featureName}">
```

Comment:
I suspect that this would need to be changed for the final integration? What about adding some "comment" to track these changes and ensure that we don't forget about them?